src/apis/tmdb.tsx

```diff
@@ -0,0 +1,24 @@
+export const fetchReleaseDate = async (movieId: number) => {
+    const apiKey = process.env.REACT_APP_API_KEY;
+    const movieUrl = process.env.REACT_APP_TMDB_BASE_URL;
+    const url = `${movieUrl}/${movieId}/release_dates?api_key=${apiKey}`;
+
+    try {
+        const response = await fetch(url);
+        if (!response.ok) {
+            throw new Error('Failed to fetch release dates');
+        }
+        const data = await response.json();
+        const releaseDates = data.results;
+        const koreanRelease = releaseDates.find(
+            (release: { iso_3166_1: string }) => release.iso_3166_1 === 'KR',
+        );
+        return koreanRelease?.release_dates[0].release_date || null;
```

Comment:
자바스크립트의 구조분해할당을 사용하시면 좋습니다.

https://ko.javascript.info/destructuring-assignment

```suggestion
        const { results: releaseDates } = await response.json();
        const koreanRelease = releaseDates.find(
            (release: { iso_3166_1: string }) => release.iso_3166_1 === 'KR',
        );
        return koreanRelease?.release_dates[0].release_date || null;
```

Reply:
return koreanRelease?.release_dates[0].release_date || null;

어차피 koreanRelease가 없으면 undefined일텐데 굳이 null을 또 다시 정의해서 넘겨줄 필요는 없겠죠?